### PR TITLE
Let Cargo Borgs break bad

### PR DESCRIPTION
**Pull request by Gr33d-y**
All this PR does is add a boxcutter to cargo borgs. Why? Because at the moment they are unable to open wrapped packages at all. This is not a massive deal on its own, however, cargoborgs are required to wrap packages to lift and haul them. This means that they are unable to deliver the packages that they wrap up, open them, and bring back the empty box in one trip, just as an example of one of the reasons they need a boxcutter.

Will this give cargoborgs a means to deal damage to others? Sure, but a lot of borgs have the ability to do damage with one or more of their tools far better than a boxcutter.

This was tested and I even learned that by doing this, borgs can recycle some wrapping paper this way! yippee!

### CODE REVIEW

**Comment by zeroisthebiggay** [comment]
nonmobular

**Comment by Zirok-BYOND** [comment]
Not sure what's up with the PR title

Change is totally fine, this has force 10(= damage, as far as I understand)+ sharpness, extinguisher which they already get has force 10 too.

Needs a comment //BUBBER EDIT ADDITION, if you could enable maintainer edits that would be swell!

**Comment by zeroisthebiggay** [comment]
> Not sure what's up with the PR title

401-boxcutter

it's a breaking bad reference

**Comment by Gr33d-y** [comment]
> Not sure what's up with the PR title
> 
> Change is totally fine, this has force 10(= damage, as far as I understand)+ sharpness, extinguisher which they already get has force 10 too.
> 
> Needs a comment //BUBBER EDIT ADDITION, if you could enable maintainer edits that would be swell!

should be enabled, does it work?

**Comment by zeroisthebiggay** [comment]
> should be enabled, does it work?

if you made changes then you didn't push or forgot to commit because they're not here

File: modular_skyrat/modules/cargoborg/code/robot_modules.dm [hunk 1]
```diff
@@ -16,6 +16,7 @@
 		/obj/item/crowbar/cyborg,
 		/obj/item/extinguisher,
 		/obj/item/universal_scanner,
+		/obj/item/boxcutter,
```
Comment (projectkepler-ru) [inline modular_skyrat/modules/cargoborg/code/robot_modules.dm:19]:
```suggestion
		/obj/item/boxcutter, //Bubberstation Edit
```

**Comment by nevimer** [comment]
im merging this right now to follow up with an upstream merge

**Pull request merged**